content/en/docs/concepts/storage/volumes.md

```diff
@@ -996,20 +996,23 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
 
 {{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
-The `CSIMigration` feature for `vsphereVolume`, when enabled, redirects all plugin operations
-from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver. In order to use this feature, the
+The `CSIMigrationvSphere` feature for `vsphereVolume` is enabled by default as of Kubernetes v1.25.
+All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver unless feature gates `CSIMigration` or `CSIMigrationvSphere` are disabled.
+
+
 [vSphere CSI driver](https://github.com/kubernetes-sigs/vsphere-csi-driver)
-must be installed on the cluster and the `CSIMigration` and `CSIMigrationvSphere`
-[feature gates](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled.
-You can find additional advice on how to migrate in VMware's
+must be installed on the cluster. You can find additional advice on how to migrate in-tree `vsphereVolume` in VMware's
 documentation page [Migrating In-Tree vSphere Volumes to vSphere Container Storage Plug-in](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-968D421F-D464-4E22-8127-6CB9FF54423F.html).
 
 Kubernetes v{{< skew currentVersion >}} requires that you are using vSphere 7.0u2 or later
 in order to migrate to the out-of-tree CSI driver.
+
+vSphere releases less than 7.0u2 are not supported for in-tree vSphere volume as of Kubernetes v1.25. 
+Please consider upgrading vSphere (both ESXi and vCenter)  to 7.0u2 or above.
```

Comment:
```suggestion
As of Kubernetes v1.25, vSphere releases less than 7.0u2 are not supported for the
(deprecated) in-tree vSphere storage driver. You must run vSphere 7.0u2 or later
in order to either continue using the deprecated driver, or to migrate to
the replacement CSI driver.
```

This clarifies that the in-tree driver is deprecated, and ties in to the following paragraph about reading the right docs for the version of Kubernetes you're actually running.